apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -244,17 +332,16 @@ export class TeacherFeedback extends Component {
       'performanceLevel4'
     ];
 
+    const studentSawFeedback =
+      this.state.latestFeedback.length > 0 &&
+      !!latestFeedback.student_seen_feedback;
```

Comment:
Will `latestFeedback.student_seen_feedback` ever be true when `this.state.latestFeedback.length` is zero or undefined? i.e. it seems like you could just use `latestFeedback.student_seen_feedback` here.

Reply:
The `latestFeedback` method can return null when length is zero or undefined. Invoking `.student_seen_feedback` on null would throw an error. I could be wrong, but I just wanted to explain my thinking. Let me know what you think :)
```
  latestFeedback = () => {
    const latestFeedback =
      this.state.latestFeedback.length > 0
        ? this.state.latestFeedback[0]
        : null;

    return latestFeedback;
  };
```

Reply:
Reading this one last time: if the condition in `latestFeedback()` ever changes so it returns null for a different reason, would this check still do what it's supposed to do? If not, I'd recommend this for future-proofing:
`const studentSawFeedback = latestFeedback && !!latestFeedback.student_seen_feedback`
Will that still accomplish what's intended?